src/components/sections/Header.tsx

```diff
@@ -66,16 +66,7 @@ export default function Header() {
             </Link>
           </div>
 
-          {toggleButton()}
-
-          <Button
-            variant="ghost" 
-            size="icon"
-            className="md:hidden hover:bg-white/10 dark:hover:bg-black/10 transition-colors"
-            onClick={() => setIsMenuOpen(!isMenuOpen)}
-          >
-            <ChevronDown className={`h-5 w-5 transition-transform duration-200 ${isMenuOpen ? 'rotate-180' : ''}`} />
-          </Button>
+          <ToggleButton/>
```

Comment:
the <Button/> you removed is not the toggle button but a button to open and close the menu in mobile devices. These changes are not appropriate.